resources/resincli.robot

```diff
@@ -151,6 +155,15 @@ Check if "${option}" variable "${variable_name}" with value "${variable_value}"
     ${result} =  Run Process    echo "${result_env.stdout}" | grep ${variable_name} | grep " ${variable_value}"    shell=yes
     Process ${result}
 
+Check if "${option}" variable "${variable_name}" with value "${variable_value}" exists on "${type}" "${id}"
+    [Documentation]    Available values for argument ${option} are: ENV, CONFIG
+    @{list} =  Create List    ENV    CONFIG
+    Should Contain    ${list}    ${option}
+    ${command}=    Set Variable If    '${option}' == 'ENV'    balena envs --${type} ${id} --json \| jq "any(.[]; .name == \"${variable_name}\" and .value == \"${variable_value}\")" \| grep true
```

Comment:
@klutchell pretty sure the `|` character doesn't need escaping.

Reply:
the quotes also don't need escaping, as existing code shows.

Reply:
it could be `==`, since it's a reserved thing in RF.

Reply:
https://balena-tests-s3-logs-bucket-1f454wud1z2ll.s3.amazonaws.com/8085964659--2869-2-b8cd086d7c3ab26fb06a3237ebcd51/logs/log.html

🎉

Reply:
LGTM, merge it!